src/domain/auth/userManager.ts

```diff
@@ -8,9 +8,11 @@ const location = `${window.location.protocol}//${window.location.hostname}${
 /* eslint-disable @typescript-eslint/camelcase */
 const settings: UserManagerSettings = {
   authority: process.env.REACT_APP_OIDC_AUTHORITY,
+  automaticSilentRenew: true,
   client_id: process.env.REACT_APP_OIDC_CLIENT_ID,
   redirect_uri: `${location}/callback`,
   response_type: 'id_token token',
+  silent_redirect_uri: `${location}/silent_renew`,
```

Comment:
We need to find a way to easily add this to tunnistamo's callback url list/add it to the readme